pkg/asset/ignition/bootstrap.go

```diff
@@ -0,0 +1,308 @@
+package ignition
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"path"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/apparentlymart/go-cidr/cidr"
+	"github.com/coreos/ignition/config/util"
+	ignition "github.com/coreos/ignition/config/v2_2/types"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/ignition/templates"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/openshift/installer/pkg/asset/kubeconfig"
+	"github.com/openshift/installer/pkg/asset/tls"
+	"github.com/openshift/installer/pkg/types"
+)
+
+const (
+	// tlsCertDirectory is the directory on the bootstrap node to place the TLS
+	// assets.
+	tlsCertDirectory = "/opt/tectonic/tls"
+)
+
+// bootstrapTemplateData is the data to use to replace values in bootstrap
+// template files.
+type bootstrapTemplateData struct {
+	KubeletImageURL            string
+	KubeletImageTag            string
+	ClusterDNSIP               string
+	CloudProvider              string
+	CloudProviderConfig        string
+	DebugConfig                string
+	KubeCoreRenderImage        string
+	MachineConfigOperatorImage string
+	EtcdCertSignerImage        string
+	EtcdctlImage               string
+	BootkubeImage              string
+	HyperkubeImage             string
```

Comment:
Why don't we just use the `HyperkubeImage` instead of `KubeletImageURL` and `KubeletImageTag` ?

Reply:
Actually `KubeletImage*` are not being used.

Reply:
It's actually not used anywhere... I thought Go warned about unused stuff.

Reply:
https://golang.org/pkg/text/template/#Template.Option